neo/core/epoch.py

```diff
@@ -91,6 +91,8 @@ def __new__(cls, times=None, durations=None, labels=None, units=None, name=None,
             times = np.array([]) * pq.s
         elif isinstance(times, (list, tuple)):
             times = np.array(times)
+        if len(times.shape) > 1:
```

Comment:
I think this could lead to unexpected behaviour. Wouldn't it be better to raise an exception if the array is not 1D?

Reply:
Yes, you are right, that this is a 'magical' conversion not visible to the user and might therefore be unexpected. However, I think that raising an exception here instead of doing the conversion might break existing code if there are additional, empty dimensions.
For now I will add an exception here.

Reply:
I see your point. Since we're not yet at version 1.0 I think it's ok to just add the exception now. In future, we should probably add a deprecation warning in one major release then add the exception in the next one.